Restructure files

HakanSunay opened the pull request:
Mimic MVC directory structure. Use absolute paths.

Commit:
Restructure project files

VladimirAnaniev committed:
use absolute paths

VladimirAnaniev commented on README.md:
```diff
@@ -1,3 +0,0 @@
-# fmi-parking
```
Think we should keep the readme

VladimirAnaniev commented on models/Car.php:
```diff
@@ -0,0 +1,7 @@
+class Car
+{
+
+}
```
Don't we need to close the php tag?

VladimirAnaniev commented:
:ship:

Commit:
Resolve merge conflicts.

Conflicts:
	src/header.php
	src/index.php
	static/images/closedbarier.jpg
	static/images/fmi.jpg
	static/images/openbarier.jpg
	styles/form.css
	styles/images.css
	styles/navbar.css
	styles/radioButton.css
	styles/style.css
	vendor/phpqrcode-2010100721_1.1.4.zip
	views/addcourse.php
	views/changePass.php
	views/changestatus.php
	views/checkcode.php
	views/forgottenPass.php
	views/login.php
	views/register.php

Commit:
Add change password view
